pkg/aws/iam.go

```diff
@@ -190,3 +196,27 @@ func unMarshallPolicyDocument(document string) (*PolicyDocument, error) {
 	}
 	return &policyDocument, nil
 }
+
+// getIAMRole returns the role object struct that has attributes of a given role
+func getIAMRole(ctx context.Context, client IAMClient, roleName string) (output *iam.GetRoleOutput, err error) {
+	input := &iam.GetRoleInput{
+		RoleName: &roleName,
+	}
+
+	output, err = client.GetRole(ctx, input)
+	if err != nil {
+		return output, err
```

Comment:
**Nit:** A more idiomatic way to do this is to just have the `return` statement. Since you're using named returns, this implies the same thing.
```suggestion
		return
```

Reply:
done